packages/theming/src/utils/arrowStyles.ts

```diff
@@ -150,45 +120,47 @@ const positionStyles = (position: ArrowPosition, size: string, inset: string) =>
  * @component
  */
 export default function arrowStyles(position: ArrowPosition, options: ArrowOptions = {}) {
-  const size = options.size || '6px';
   const inset = options.inset || '0';
-  const squareSize = math(`${size} * 2 / sqrt(2)`, exponentialSymbols);
+  const size = options.size === undefined ? 6 : (stripUnit(options.size) as number);
+  const squareSize = `${Math.round((size * 2) / Math.sqrt(2))}px`;
```

Comment:
I'm glad this works. I was having [problems](https://github.com/zendeskgarden/react-components/pull/1812/files#diff-e8a3ada410dbb11bc3294a80df7fd05ac795d41d99ffdc21e4597c7787fa7991R149) and needed `Math.floor`.